<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QWidget" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineUsername">
     <property name="placeholderText">
      <string>Username</string>
     </property>
     <property name="minimumHeight">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="linePassword">
     <property name="placeholderText">
      <string>Password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="minimumHeight">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLogin">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="minimumHeight">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>